rtsp.go

```diff
@@ -404,12 +404,12 @@ func (rc *rtspCamera) initH264(session *description.Session) (err error) {
 		}
 
 		publishToWebRTC := func(pkt *rtp.Packet) {
-			pts, ok := rc.client.PacketPTS(media, pkt)
+			pts, ok := rc.client.PacketPTS2(media, pkt)
 			if !ok {
 				return
 			}
 			ntp := time.Now()
-			u, err := fp.ProcessRTPPacket(pkt, ntp, pts, true)
+			u, err := fp.ProcessRTPPacket(pkt, ntp, time.Duration(pts), true)
```

Comment:
Is PTS not time since epoch?

Reply:
It is not necessarily time since epoch but normally time since the start of the media stream. In this case we are simply parsing out the PTS from the RTP packet we are receiving.